### PR TITLE
Homework uploading for assignment 01

### PR DESCRIPTION
**Pull request by eburkey**
@samanthashepard
@lendijoy

### CODE REVIEW

File: HW01_PalmerPenguins/HW01.Rmd [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: "HW01.Rmd"
+author: "Elena Burkey"
+date: "1/25/2021"
+output: html_document
```
Comment (lendijoy) [inline HW01_PalmerPenguins/HW01.Rmd:5]: Make sure to also render your output to github format. Including "keep_md: true" below the output line works.

File: HW01_PalmerPenguins/HW01.Rmd [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(palmerpenguins)
+summary(penguins)
+```
+
+## **Summary**
+
+### **General Structure:**
+The data set of the penguins have bee organized into 344 rows and 8 columns whereas the penguins raw data has been organized into 344 rows and 17 columns. There are 165 female and 168 male penguins that have been used in the data set.
```
Comment (lendijoy) [inline HW01_PalmerPenguins/HW01.Rmd:17]: Typo: "has been"

File: HW01_PalmerPenguins/HW01.Rmd [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(palmerpenguins)
+summary(penguins)
+```
+
+## **Summary**
+
+### **General Structure:**
```
Comment (lendijoy) [inline HW01_PalmerPenguins/HW01.Rmd:16]: Don't forget to include your code and output. Remember the assignment is to use at least three functions and showcase the dataset.

File: HW01_PalmerPenguins/HW01.Rmd [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+Flipper Length
+Mean:*200.9 mm*
+Median: *197.0 mm*
+
+Body Mass
+Mean: *4202 g*
+Median: *4050 g*
+
+```{r penguins}
+summary(penguins)
```
Comment (lendijoy) [inline HW01_PalmerPenguins/HW01.Rmd:38]: I noticed that you also included this line above. Did you add or change something that you are showing here?

**Review by lendijoy — CHANGES_REQUESTED** [review]
Good start! Just a few things to add/clarify.

File: HW01_PalmerPenguins/HW01.Rmd [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+```{r plot, echo=FALSE}
+plot(penguins$bill_length_mm)
```
Comment (samanthashepard) [inline HW01_PalmerPenguins/HW01.Rmd:42]: I'm unclear on what the x-axis is supposed to represent?

File: HW01_PalmerPenguins/HW01.Rmd [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
+title: "HW01.Rmd"
+author: "Elena Burkey"
+date: "1/25/2021"
+output: html_document
+---
```
Comment (samanthashepard) [inline HW01_PalmerPenguins/HW01.Rmd:6]: I ran the code and it seems to be working fine!

File: HW01_PalmerPenguins/HW01.Rmd [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+```
+
+## **Summary**
+
+### **General Structure:**
+The data set of the penguins have bee organized into 344 rows and 8 columns whereas the penguins raw data has been organized into 344 rows and 17 columns. There are 165 female and 168 male penguins that have been used in the data set. 
+
+### **Descriptive Statistics:** 
+#### Physical Characteristics
+Bill length
```
Comment (samanthashepard) [inline HW01_PalmerPenguins/HW01.Rmd:21]: I don't think it's very useful to include this, when this info is already in the table? Maybe there's a way to bold specific statistics in the output (e.g., means and medians)

File: HW01_PalmerPenguins/HW01.Rmd (hunk 3 above)
Comment (samanthashepard) [reply HW01_PalmerPenguins/HW01.Rmd:16]: Question: what do you mean by including the output? Upload a separate file?

Comment (samanthashepard) [reply HW01_PalmerPenguins/HW01.Rmd:16]: Oh, output = the github document, nvm! I thought they were different things.

**Pull request merged**